### DISCUSSION TITLE
`@fastify/session` vs `@fastify/secure-session`? Which to use? and when?

### DISCUSSION THREAD

**Discussion by jessekrubin** [fastify/fastify#4186] [Q&A]
I am just curious which fastify session plugin people recommend using between `@fastify/session` and `@fastify/secure-session`? and why?

**Comment by TheWashiba** [comment]
Well, that's a weird question to begin with. "What do I need?" is the question you should ask. Their descriptions give out the purpose quite quickly, one is for server side state, another one actually stores everything on the client side, hence the "stateless" keyword. I guess Fastify could unify those session libraries in the future.
